rq/serializers.py

```diff
@@ -0,0 +1,80 @@
+from abc import ABCMeta
+from functools import partial
+from .exceptions import UnpickleError, DeserializationError
+
+try:
+    import cPickle as pickle
+except ImportError:  # noqa  # pragma: no cover
+    import pickle
+
+
+class BaseSerializer(object):
+    __metaclass__ = ABCMeta
+
+    def dumps(self, value):
+        raise NotImplementedError('This method must be implemented')
+
+    def loads(self, value):
+        raise NotImplementedError('This method must be implemented')
+
+
+class DefaultSerializer(BaseSerializer):
```

Comment:
I don't think we need this class. Our default serializer is just `pickle`

Reply:
I agree. 
Also remember that our default serializer `pickle` also implements

```
        try:
            serialized_data = partial(pickle.dumps, protocol=pickle.HIGHEST_PROTOCOL)(data)
        except Exception as e:
            raise UnpickleError('Unpickleable return value', data, e)
```

Reply:
Would this not be backwards compatible in certain instances? One scenario that comes to mind would be if you have workers in different Python versions with different pickling protocols.